src/RefractAPI.cc

```diff
@@ -6,7 +6,6 @@
 //  Copyright (c) 2015 Apiary Inc. All rights reserved.
 //
 
-#include <iterator>
```

Comment:
Curious, any particular reason you've moved this down? In most languages it preferred to put system includes before local includes.